API/SSW.Rewards.Persistence/Migrations/20220607061234_UnFuckDatabase.cs

```diff
@@ -3,10 +3,51 @@
 
 namespace SSW.Rewards.Persistence.Migrations
 {
-    public partial class AddRoles : Migration
+    public partial class UnFuckDatabase : Migration
```

Comment:
Are we merging this?

Please rename this migration!

Reply:
Yes, please change this :)

Reply:
resolved